depinject/invoke_test.go

```diff
@@ -0,0 +1,71 @@
+package depinject_test
+
+import (
+	"testing"
+
+	"github.com/regen-network/gocuke"
+	"gotest.tools/v3/assert"
```

Comment:
nit: I've noticed some tests use this package instead of testify. I think it makes more sense to generalize everything to use testify.

Reply:
I actually think we should use gotest.tools instead of testify wherever possible. There are longstanding issues with testify in particular https://github.com/stretchr/testify/issues/535 which is a deal breaker for comparing protobuf values (with the new API) and https://github.com/stretchr/testify/issues/187 (testify suites should be considered an anti-pattern because of this). Basically testify is bloated and stuck in the past. gotest.tools is minimal and correct and also comes with useful support for golden files

Reply:
I didn't know, interesting!

Reply:
can we open an issue for this? the migration can be a good first issue

Reply:
I have created https://github.com/cosmos/cosmos-sdk/issues/12332, feel free to edit it.